Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -3119,12 +3119,28 @@ void CGContextDrawTiledImage(CGContextRef context, CGRect rect, CGImageRef image
 #pragma endregion
 
 #pragma region Drawing Operations - Gradient + Shading
+
+static bool _isFloatEqual(const float a, const float b) {
```

Comment:
since we can have a float lol, but other than that, i agree.

---
In reply to: [107500426](https://github.com/Microsoft/WinObjC/pull/2249#discussion_r107500426) [](ancestors = 107500426)